code/.eslintrc.json

```diff
@@ -1,7 +1,5 @@
 {
-  "extends": [
-    "airbnb"
-  ],
+  "extends": ["airbnb"],
```

Comment:
Was this just "tidying up" or does it help with the code? Honest question, anything that helps is good to know! :-D

Reply:
I dont even remember doing this 😅 I think it might be the result of some eslint/prettier config?🤔